packages/plugins/rubygem-newt/rubygem-newt.spec

```diff
@@ -74,24 +68,28 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-mkdir -p %{buildroot}%{gem_extdir_mri_lib}/ruby_newt
-cp -a .%{gem_instdir}/ext/ruby_newt/*.so %{buildroot}%{gem_extdir_mri_lib}/ruby_newt/
+mkdir -p %{buildroot}%{gem_extdir_mri}/ruby_newt
+cp -a .%{gem_instdir}/ext/ruby_newt/*.so %{buildroot}%{gem_extdir_mri}/ruby_newt/
 touch %{buildroot}%{gem_extdir_mri}/gem.build_complete
 
 # Prevent dangling symlink in -debuginfo (rhbz#878863).
 rm -rf %{buildroot}%{gem_instdir}/{ext,tmp,.require_paths}
 
 %check
+find %{buildroot}
```

Comment:
this does not belong here I guess? :)

Reply:
Helps debugging a lot, you want this out? Won't harm...

Reply:
I'd prefer not to have it here.

Reply:
Sure, removed. Rebased.